etcdserver/util.go

```diff
@@ -99,15 +103,40 @@ func (nc *notifier) notify(err error) {
 	close(nc.c)
 }
 
-func warnOfExpensiveRequest(lg *zap.Logger, now time.Time, stringer fmt.Stringer) {
-	warnOfExpensiveGenericRequest(lg, now, stringer, "")
+func warnOfExpensiveRequest(lg *zap.Logger, now time.Time, reqStringer fmt.Stringer, respMsg proto.Message, err error) {
+	var resp string
+	if !isNil(respMsg) {
+		resp = fmt.Sprintf("size:%s", proto.Size(respMsg))
```

Comment:
`%d`?

Reply:
Yes, thanks. The safeSize function I used before returned a string. Fixing.